source/_integrations/weather.markdown

```diff
@@ -4,6 +4,7 @@ description: Instructions on how to setup your Weather platforms with Home Assis
 ha_category:
   - Weather
 ha_release: 0.32
+ha_iot_class: Cloud Polling
```

Comment:
I think we should set this to None/null.

```suggestion
ha_iot_class: ~
```

Reasoning: The weather integration, by itself, doesn't connect to any resource. It isn't usable even. This an entity component and is build upon by other weather integration.

Reply:
Thanks for the advice. I'm kind of new to open source contributing so this is very helpful.